app/views/pvb1_paths/status.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :header, 'Visit expired' %>
+
+<div class='Grid'>
+  <div class='Grid-2-3'>
+    <article>
+      <p>This visit has expired and its information has been removed.<p>
```

Comment:
It's nor so much about have a localisation as it is that all of our other copy is in en.yml.